vmdb/app/views/layouts/_perf_charts.html.haml

```diff
@@ -6,15 +6,27 @@
     - if chart_data
       - if !perf_parent? && !perf_compare_vm?
         - if charts && chart_data.length > 0
-          - (0..chart_data.length - 1).each do |c|
-            - unless charts[c][:type] == "None" || chart_data[c].nil?
-              = render(:partial    => '/layouts/perf_chart',
-                       :locals     => {:charts      => charts,
-                                       :chart_data  => chart_data,
-                                       :chart_set   => chart_set,
-                                       :chart_index => c,
-                                       :width       => width,
-                                       :height      => height})
+          .row
+            - (0..chart_data.length - 1).each_with_index do |c, idx|
+              - unless charts[c][:type] == "None" || chart_data[c].nil?
+                - if idx%2

```

Comment:
@epwinchell : I don't see any difference between the positive and negative branch here.